Formula/alluxio.rb

```diff
@@ -1,24 +1,20 @@
 class Alluxio < Formula
   desc "Open Source Memory Speed Virtual Distributed Storage"
   homepage "https://www.alluxio.io/"
-  url "https://downloads.alluxio.io/downloads/files/2.4.1/alluxio-2.4.1-bin.tar.gz"
-  sha256 "06c052761f597692a4427f08169f3277f458cf23847db4270b1f2dd75c14907a"
+  url "https://downloads.alluxio.io/downloads/files/2.5.0-2/alluxio-2.5.0-2-bin.tar.gz"
+  sha256 "3921176e5206ad278d9154b20cb304ed85ffa70cb14a53c4f9f71f1eedbf3871"
   license "Apache-2.0"
 
   livecheck do
     url "https://downloads.alluxio.io/downloads/files/"
-    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
+    regex(%r{href=["']?v?(\d+(?:[.-]\d+)+)/?["' >]}i)
   end
 
   bottle :unneeded
```

Comment:
```suggestion
```
I think we can try to bottle the formula in this PR (towards #75943).

Reply:
OK, though I note that the bottles that will be generated are:
* huge (1.8GB each) and
* almost certainly identical across platforms (the only executable assets are JARs and shell scripts)

Reply:
> * huge (1.8GB each)

That's ok but why are they so big?

> * almost certainly identical across platforms (the only executable assets are JARs and shell scripts)

Yes, we have a `all` bottle now for this.

Reply:
> why are they so big?

The vast bulk of it (1.6GB) is just JARs, 236 of them:
```
$ find * -name \*.jar | xargs du -ck | sort -n | nl | tail -5
   233	100352	libexec/lib/alluxio-table-server-underdb-hive-2.5.0-2.jar
   234	105472	libexec/assembly/alluxio-client-2.5.0-2.jar
   235	132096	libexec/lib/alluxio-integration-tools-hms-2.5.0-2.jar
   236	164864	libexec/assembly/alluxio-server-2.5.0-2.jar
   237	1626248	total
```
I don't know any more than that. I raised this PR only because I noticed that `alluxio` and some other FUSE formulae were out-of-date while migrating them to `gromgit/fuse`.